### ISSUE TITLE
[BUG] <Select> Select shows active border when disabled.

### ISSUE THREAD

**Issue by DaiQiangReal** [DouyinFE/semi-design#949]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Which Component

select

### Semi Version

latest

### Current Behavior

Click a disabled select shows an active border.

### Expected Behavior

Click a disabled select won't show anything.

### Steps To Reproduce

Click a disabled select.

### ReproducibleCode

```markdown
<Select disabled defaultValue='abc' style={{ width: 120 }}>
            <Select.Option value='abc'>抖音</Select.Option>
            <Select.Option value='hotsoon'>火山</Select.Option>
            <Select.Option value='jianying' disabled>剪映</Select.Option>
            <Select.Option value='xigua'>西瓜视频</Select.Option>
        </Select>
```

### Environment

```markdown
- OS:
- browser:
```

**Comment by shijiatongxue** [comment]
https://github.com/DouyinFE/semi-design/pull/570

![image](https://user-images.githubusercontent.com/26477537/177695389-10231508-029f-4258-a174-af0d47fca962.png)

**Comment by pointhalo** [comment]
fix after v2.17
